### ISSUE TITLE
FOOOF FAQs: collecting & providing responses to common questions

### ISSUE THREAD

**Issue by TomDonoghue** [fooof-tools/fooof#95]
There are some relatively common questions that come up, that could be addressed in general. Perhaps these are things that could also have more time in the tutorials, but for more of a 'Quick-Access' perspective, I think we can/should collect an FAQ (and this becomes easy to host now when using Sphinx). 

Collection of questions / answers to include (answers are not complete):
- I'm interested in a particular oscillation band, should I fit a small range?
    - Generally, no, always try and fit a broad range
- So what is slope anyways?
    - Point to Gao et al, 2017
- How do I pick settings?
   - Quick version: start at best guess, check plots and go from there [Note: recommend 'train / test' split]
- What data does this work on?
    - Basically, any power spectra from neuro-electrophysiological field data
- How do I make the power spectra?
    - Doesn't really matter. 
- Does how I make the power spectra matter?
    - Probably. Perhaps a note on frequency resolution?
- Why is this different from prior things / what makes it work?
    - Broadly: by jointly learning aperiodic & periodic, and in particular, gains in slope fitting (otherwise: once one thing goes awry, everything else fails apart with it. )

**Comment by TomDonoghue** [comment]
Okay - I pushed an update to the FAQ, the source file for which is here:
https://github.com/fooof-tools/fooof/blob/master/doc/faq.rst

It's also pushed to the doc site, here:
https://fooof-tools.github.io/fooof/faq.html

As often for my 'first-draft' like writing, it's perhaps overly verbose. I also tried to add a few things that contextualize and explain our naming schemes, and highlight / motivate a bit how we are thinking about these ideas. 

@voytek & @rdgao when you get a chance, could you check through the FAQs and 'review' them? In particular:
- Do you agree with all the answers? Everything seem reasonable, etc?
- Are there any other questions that come up a lot, that should be added?

Let me know of any suggested edits on this issue, or submit a PR if you want to dive into it .

**Comment by rdgao** [comment]
I'll just leave comments here:

> By 'aperiodic' activity we mean the arythmic component of the signal, that typically has a 1/f like distribution, whereby the power systematically decreases with increasing frequencies."
- I tend to avoid using "distribution" here and use "follows a 1/f-like function", because it also gets used in the context of probability distributions, particularly in scale-free conversations. That's just a personal preference though.

- I really like the aperiodic explanation, in lieu of background/noise/whatever
- typo at "what we know call aperiodic..."
- typo in various places where (ar)rhythmic is spelled (a)rythmic

> Though we often focus on the rhythmic (periodic) components, neurophysiological recordings of electromagnetic activity in the brain are fundamentally composed of aperiodic activity upon which there is (sometimes) also rythmic components.
- not to open this can of worms again here, but I would at least state that this is a presumed model we have (with citations), in that the aperiodic and periodic components ARE separable "components" of the signal itself, not just in the PSD. This is agnostic to whether they have different physiological interpretations, or at all, as you get into in the later section. 

- typo "Many existing approaches do no attempt" 
- typo "one difference is that FOOOF does not prioritize one of the other component" one OR the other?
- typo "Generally, no, it is better to always try and fit a broad range, rather then to fit a small frequency range" should be than 
- typo " it doesn't matter exactly how you calculate power spectra that you will fit with FOOOF, for example, using different methdos" methods

other than those, looks good to me! Pretty comprehensive set of answers, I haven't come across any other common questions, I don't think.

**Comment by TomDonoghue** [comment]
Agreed on both! 

Very important to be explicit that FOOOF is a model driven approach, and as such is very 'opinionated' about how to go about measuring activity. Also, yeh, we should point out we don't have strong assumptions about oscillations (not implying they are continuous / they are likely to be bursty, etc). 

I'll fold all these comments into a new PR that finishes up a bunch of the sphinx stuff, after the current PRs go through.

**Comment by TomDonoghue** [comment]
Updates pushed to sphinx branch - should address all these if you want to have a look. 

Citations:
- @rdgao any particular citations in mind for the 'model with two components of the spectrum' thing. I added a note of that at the top, but not sure what the most obvious thing to cite is. 
- Is it the Buzsaki paper that first defines oscillations as power above the 1/f? We could maybe cite that too, in reference to how we operationally define periodic. 

Also, I think I'll add a question / answer about interpreting peaks, and link out to ByCycle, or something about shape. 

Sidenote: I think I've been spelling 'rhythm' and 'arrhythmic' wrong, like forever. Who knew there were so many 'h's and 'r's???

**Comment by voytek** [comment]
I usually cite these for "bumps above 1/f":

“[when] particular oscillation frequencies become dominant… a peak (bump) appears in the respective frequency band.” - Buzsáki el al., Neuron 2013

“In a power spectrum, brain oscillations appear as bumps on top of this 1/f slope…” - He B, Trends Cogn Sci 2014

**Comment by TomDonoghue** [comment]
I left this open because there was these last couple references I was going to add. I've gone through and added a couple more references, and done some updates / clean ups of the FAQ, now added to #152. Closing this now, unless there is any more discussion needed of FAQ content.
